### PR TITLE
style: clean up linter complaints

### PR DESCRIPTION
**Pull request by smoyer64**

### CODE REVIEW

File: commands/bug/bug_select.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ The complementary command is "git bug deselect" performing the opposite operatio
 
 func runBugSelect(env *execenv.Env, args []string) error {
 	if len(args) == 0 {
-		return errors.New("You must provide a bug id")
+		return errors.New("a bug id must be provided")
```
Comment (MichaelMure) [inline commands/bug/bug_select.go:47]: I've been guilty of making those error messages a bit more human as they show up directly to the user. That was a long time ago though, dunno if it really makes sense.

File: commands/bug/bug_show.go [hunk 2]
```diff
@@ -91,7 +91,9 @@ func runBugShow(env *execenv.Env, opts bugShowOptions, args []string) error {
 		case "title":
 			env.Out.Printf("%s\n", snap.Title)
 		default:
-			return fmt.Errorf("\nUnsupported field: %s\n", opts.fields)
+			env.Out.Println()
```
Comment (MichaelMure) [inline commands/bug/bug_show.go:94]: trying out that command, it doesn't make that much sense to have a new line, I think we can remove it entirely.

File: commands/bug/bug_comment.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ func newBugCommentCommand(env *execenv.Env) *cobra.Command {
 }
 
 func runBugComment(env *execenv.Env, args []string) error {
-	b, args, err := ResolveSelected(env.Backend, args)
+	b, _, err := ResolveSelected(env.Backend, args)
```
[the remark this answers is not in the document]
Comment (smoyer64) [reply commands/bug/bug_comment.go:29]: And that pattern continues to exist where the `args` is actually used ... the functions I changed are the ones where the `args` are never used.

File: commands/bug/bug_select.go (hunk 1 above)
Comment (smoyer64) [reply commands/bug/bug_select.go:47]: I tried to make it somewhat human and match the convention for error messages - your choice (the capitalization was what the linter complained about and I rearranged it to get rid of the word "You")

File: commands/bug/bug_show.go (hunk 2 above)
Comment (smoyer64) [reply commands/bug/bug_show.go:94]: I removed one of the two new-lines ... I thought maybe that was to separate an error output from the field output that might be directly above it.

Comment (MichaelMure) [reply commands/bug/bug_show.go:94]: Let's remove both.

File: commands/bug/bug_select.go (hunk 1 above)
Comment (MichaelMure) [reply commands/bug/bug_select.go:47]: Yeah, that also make sense. I suppose that to make things really proper we would need some kind of augmented error to render exactly how it should be, which would also be required  if we ever do internationalization.

File: commands/bug/bug_comment.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (smoyer64) [reply commands/bug/bug_comment.go:29]: So part of the solution should be to not shadow the function parameter that's already named `args`?

[the remark this answers is not in the document]
Comment (MichaelMure) [reply commands/bug/bug_comment.go:29]: heh, I don't see an optimal solution that also make the linter happy. But that's alright.

**Pull request merged**